src/Workspaces/VisualBasic/Portable/Indentation/VisualBasicIndentationService.Indenter.vb

```diff
@@ -63,20 +68,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Indentation
                     Return GetIndentationBasedOnToken(indenter, GetTokenOnLeft(trivia), trivia)
                 End If
 
-                ' okay, now check whether the trivia is at the beginning of the line
-                Dim firstNonWhitespacePosition = previousLine.GetFirstNonWhitespacePosition()
-                If Not firstNonWhitespacePosition.HasValue Then
-                    Return indenter.IndentFromStartOfLine(0)
-                End If
-
-                Dim firstTokenOnLine = indenter.Root.FindToken(firstNonWhitespacePosition.Value, findInsideTrivia:=True)
-                If firstTokenOnLine.Kind <> SyntaxKind.None AndAlso firstTokenOnLine.Span.Contains(firstNonWhitespacePosition.Value) Then
-                    'okay, beginning of the line is not trivia, use this token as the base token
-                    Return GetIndentationBasedOnToken(indenter, firstTokenOnLine)
+                If trivia.Kind = SyntaxKind.LineContinuationTrivia Then
+                    Return GetIndentationBasedOnToken(indenter, GetTokenOnLeft(trivia), trivia)
                 End If
+            End If
 
-                Return indenter.GetIndentationOfLine(previousLine)
+            If tokenOpt.HasValue Then
+                Return GetIndentationBasedOnToken(indenter, tokenOpt.Value)
             End If
+
```

Comment:
the rest of the logic fell out by just properly grabbing the real preceding trivia/token.